doc/nrf/protocols/matter/getting_started/low_power_configuration.rst

```diff
@@ -19,6 +19,65 @@ The following Matter samples and applications use the low power configuration by
 * :ref:`Matter window covering sample <matter_window_covering_sample>`
 * :ref:`Matter weather station application <matter_weather_station_app>`
 
+.. _ug_matter_device_low_power_icd:
+
+Enable Matter Intermittently Connected Devices support
+******************************************************
+
+The Matter treats power-optimized devices as a part of the wider group, which are devices that are intermittently connected to the network or unreachable for periods of time.
```

Comment:
Combining this with mine:
```suggestion
In Matter, power-optimized devices are part of the wider group of devices which includes devices that are intermittently connected to the network or otherwise unreachable for periods of time.
```